tests/generic_views/test_edit.py

```diff
@@ -90,6 +90,21 @@ def test_post_data(self):
         self.assertRedirects(res, '/list/authors/')
 
 
+@override_settings(ROOT_URLCONF='generic_views.urls')
```

Comment:
A couple of notes:

I'm not sure how to add urls in a separate module, - import urlconf object and add url to it? 

I just want to be clear about this distinction: all of the tests there are for gCBVs that django provides, and the intent is "make sure views we provide work as expected". Combined views are not provided by django, and there is nearly unlimited number of possible combinations, with various methods overridden, some of them may work and others will not work. So, the intent of this test is quite different: 'test that this specific update allows an easier way to combine views.'

However we currently don't document combined views nor this specific way of combining views, so  if this test breaks in the future, it's probably ok because we're not committing officially to this anywhere (but I may have that in the new docs I've been working on and we talked about recently).